src/actions/images/images.js

```diff
@@ -0,0 +1,12 @@
+import { imagesTypes } from '../../actions-types';
```

Comment:
@djallas, same thing here, it would also be good if you separate actions by file